packages/frontend-web/src/app/util/dataService.ts

```diff
@@ -750,9 +750,11 @@ export async function destroyRelationshipModels(
 ): Promise<void> {
   validateModelName(type);
 
+  // TODO: Test this.  Delete might not like that arguments are passed in the body
```

Comment:
My referred way to have TODOs is to have TODO(issue number): ... That helps make links TODOs and issue.